.circleci/config.yml

```diff
@@ -840,6 +840,7 @@ jobs:
       EMTEST_SKIP_SIMD: "1"
       EMTEST_SKIP_SCONS: "1"
       EMTEST_SKIP_NODE_CANARY: "1"
+      EMTEST_BROWSER: "0"
```

Comment:
Why is this now needed?

Reply:
Without this line the attempt to run `sockets.test_nodejs_sockets_echo` would launch a browser (because test_sockets.py is a browser test suite).

Reply:
Hmm, but why is this needed now, and not before this PR?